src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private int side;
+
+    public Square(String color, int side) {
+        super(color);
+        this.side = side;
+    }
+
+    public int getSide() {
+        return side;
+    }

```

Comment:
you don't need getters for all Figure implementations in this task, remove them please

Reply:
Done